android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
+    private String endsAt;
     private String timezone;
     private String description;
-    @JsonProperty("logo-url")
-    private String logo;
-    @JsonProperty("organizer-name")
+    private String logoUrl;
     private String organizerName;
-    @JsonProperty("organizer-description")
     private String organizerDescription;
-    @Relationship("social-links")
-    private RealmList<SocialLink> socialLinks;
-    @JsonProperty("ticket-url")
     private String ticketUrl;
     private String privacy;
     private String type;
     private String topic;
-    @JsonProperty("sub-topic")
     private String subTopic;
-    @JsonProperty("code-of-conduct")
     private String codeOfConduct;
-    @Relationship("copyright")
-    private Copyright copyright;
     private String email;
-    @JsonProperty("schedule-published-on")
     private String schedulePublishedOn;
-    @JsonProperty("searchable-location-name")
     private String searchableLocationName;
     private String state;
-    @JsonProperty("is-sessions-speakers-enabled")
-    private Boolean hasSessionSpeakers;
-    @JsonProperty("thumbnail-image-url")
-    private String thumbnail;
-    @JsonProperty("original-image-url")
-    private String backgroundImage;
-    @JsonProperty("large-image-url")
-    private String large;
-    @JsonProperty("icon-image-url")
+    private Boolean isSessionsSpeakersEnabled;
+    private String thumbnailImageUrl;
+    private String originalImageUrl;
+    private String largeImageUrl;
     private String iconImageUrl;
-    @JsonProperty("created-at")
     private String createdAt;
-    @JsonProperty("deleted-at")
     private String deletedAt;
+    @Relationship("copyright")
+    private Copyright copyright;
     @Relationship("speakers-call")
-    private CallForPapers callForPapers;
+    private CallForPapers speakersCall;
+    @Relationship("social-links")
```

Comment:
as we are changing methods according to the new attribute names, dont you think CallForPapers class be changed to SpeakersCall?